docs/source/deployguide/use_CA.md

```diff
@@ -0,0 +1,282 @@
+
+![Structuring organizations](../images/organizations.png)
+
+*The figure above shows the structure of the organizations managed by an administrator.*
+
+Even if you don't plan to create an orderer organization, this kind of structure provides the highest level of long term flexibility for your deployment. If you create a new peer, for example, you will know to create a folder at `organizations/<name of org>/<name of new peer>`. This `<name of new peer>` folder will be the location for the local MSP of the peer (generated when the peer identity is enrolled) and for the certificates generated through enrollment with the TLS CA. Similarly, the location of the MSP of the organization the peer belongs to can reference the `msp` folder of the organization (which includes both the `config.yaml` file if Node OUs are being used as well as the public certificate of the admin of the organization, which in many cases will be the admin of the peer).
+
+![Organizations and peers](../images/organizationswithpeer.png)
+
+*The figure above shows the subfolders inside of a peer owned by the organization. Note the `msp` folder here under the `peers` folder. This is the local MSP of the peer, not a duplicate of the `org1.example.com` MSP.*
+
+It is the best practice to create these folders before enrolling identities and then referencing them when issuing the enroll command through the `--mspdir` flag. Note that while the --mspdir flag is used to specify where the MSP of the CA admin is during **registration**, it is used instead during **enrollment** to specify the location on the filesystem where the folders and certificates returned by the CA will be stored.
+
+### Specifying NodeOUs
+
+In previous versions of Fabric, identities only had two types: `client` and `peer`. The `peer` type was used for both peers and ordering nodes, while the `client` type was used for both clients (applications) and admins, with the placement of a `client` type in a special `admincerts` folder making the identity an admin within a particular context.
+
+Now it is possible, and recommended, to encode not just `peer` or `client`, but also `orderer` or `admin` roles into the certificates generated by a CA using NodeOUs. Note that an identity can only have one of these roles, and that to enable these roles you must copy the relevant stanzas into a file called `config.yaml`. This `config.yaml` file is used by Fabric in different ways. In a channel MSP, it is used to verify that the `admin` of an organization has a role of `admin` (this replaces the use an `admincerts` folder which was used in older versions of Fabric). In the local MSP node, it is used not just to verify the node admin but that the node has either a `peer` or `orderer` role.
```

Comment:
In the local MSP node, it is used not just to verify the node admin but that the node has either a `peer` or `orderer` role. << I'm not sure what you mean by `it is used not just to verify the node admin`? If it has a peer or orderer role it cannot be a node admin.

Reply:
The idea here was to say that for admins the only verification done in terms of a NodeOU is for the admin. Fabric will read the config.yaml file and check that against the OU offered by the identity. Whereas with the node the OU of the node is verified in addition to the OU of the admin. 

First off, I hope that's correct. Secondly, is getting into this maybe just going to confuse people. I added a bit of clarification in the doc.